spec/helpers/options_helper_spec.rb

```diff
@@ -23,15 +23,15 @@
                         'CC-BY-NC' => 'http://creativecommons.org/licenses/by-nc/4.0/',
                         'CC-BY-NC-ND' => 'http://creativecommons.org/licenses/by-nc-nd/4.0/',
                         'CC-BY-NC-SA' => 'http://creativecommons.org/licenses/by-nc-sa/4.0/',
-                        'CC-BY-ND' => 'http://creativecommons.org/licenses/by-sa/4.0/',
-                        'CC-BY-SA' => 'http://creativecommons.org/licenses/by-nd/4.0/',
+                        'CC-BY-ND' => 'http://creativecommons.org/licenses/by-nd/4.0/',
+                        'CC-BY-SA' => 'http://creativecommons.org/licenses/by-sa/4.0/',
                         'CC0' => 'http://creativecommons.org/publicdomain/zero/1.0/',
                         'Education Use Permitted' => 'http://opaquenamespace.org/ns/rights/educational/',
-                        'Free Access - No Re-Use' => 'http://www.europeana.eu/rights/rr-f/',
+                        'Free Access - No Re-Use' => 'http://www.europeana.eu/portal/rights/rr-f.html',
```

Comment:
I thought we had trouble with that URL because it redirected?  I can change the 3 europeana URIs back the way they were.  I just wasn't sure what the correct URI was.